english/lucidream/luciddream.md

```diff
@@ -0,0 +1,17 @@
+As you see the door opens, while you well aware that the spell you just cast is
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.